electron/main.js

```diff
@@ -86,6 +86,9 @@ if (process.platform === 'win32') {
   });
 }
 
+if (shouldQuit) {
+  app.quit();
+}
```

Comment:
I would also put it inside the above `if` but again.. we'll have to investigate that more.. for example if the app is minimized.. will it show the current instance that is running when you try to open a new one?